app/html/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html ng-app="ramble">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Keep Calm and Ramble On</title>
+    <script src="bundle.js"></script>
+    <link rel="stylesheet" href="bundle.css">
+  </head>
+  <body>
+    <ng-view></ng-view>
```

Comment:
Love that yall chose to let the body of this app only include an ng-view. This makes working with angular so much easier!